### PR TITLE
Split view action bar

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: src/app/app-extras.module.ts [hunk 1]
```diff
@@ -11,17 +11,21 @@ import {
 } from '@skyux/indicators';
 
 import {
-  SkyModalModule
+  SkyModalModule, SkyConfirmModule
```
Comment (Blackbaud-AlexKingman) [inline src/app/app-extras.module.ts]: Code style > import formatting.

File: src/app/visual/split-view-summary-action-bar/split-view-summary-action-bar-demo.component.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
+<div class="app-screenshot" id="screenshot-split-view" style="max-height: 900px; height: 900px">
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/split-view-summary-action-bar/split-view-summary-action-bar-demo.component.html]: Code style > attributes should be on new lines.

File: src/app/app-extras.module.ts [hunk 3]
```diff
@@ -37,6 +41,8 @@ import {
     SkyModalModule,
     SkySummaryActionBarModule,
     SkyTabsModule,
+    SkySplitViewModule,
+    SkyConfirmModule,
```
Comment (Blackbaud-AlexKingman) [inline src/app/app-extras.module.ts]: Does your split view demo necessarily need the `SkyConfirmModule`?

File: src/app/app-extras.module.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/app-extras.module.ts]: Fixed

File: src/app/visual/split-view-summary-action-bar/split-view-summary-action-bar-demo.component.html (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/visual/split-view-summary-action-bar/split-view-summary-action-bar-demo.component.html]: Fixed

File: src/app/app-extras.module.ts (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/app-extras.module.ts]: Yes. It has a feature in it that pops a confirm modal.

**Comment by Blackbaud-AlexKingman** [comment]
The split view animations aren't working. Is that indicative of something missing?

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AlexKingman that is due to the `NoopAnimationsModule` in the app-extras. Remove that locally and they should work. It is there to ensure visual tests don’t hit a mid-animation state.

**Pull request merged**